api/docs/release.dox

```diff
@@ -139,6 +139,9 @@ Further non-compatibility-affecting changes include:
    -core_serial) and control the schedule
```

Comment:
> …cords matching operands.

As you can see, the title is truncated and wrapped around, which is not ideal.  Please tighten up the title to avoid truncation.  Maybe "i#5975: Add invariant check that read/write records match operands".

Reply:
done